package chattp;

//option optimize_for = LITE_RUNTIME;

message ChattpMessage
{
    required string sender = 1;
    required string receiver = 2; // group name if group_message.

    optional bool group_message = 3 [default = false];

    required string body = 4;

    required string timestamp = 5; // possibly RFC2822 (Mon, 16 Jun 2014 21:47:06 +0200)
}
